skl2onnx/_parse.py

```diff
@@ -337,16 +347,15 @@ def _parse_sklearn_column_transformer(scope, model, inputs):
         return transformed_result_names
 
 
-def _parse_sklearn(scope, model, inputs):
+def parse_sklearn(scope, model, inputs):
     '''
-    This is a delegate function. It doesn't nothing but invoke the correct parsing function according to the input
-    model's type.
+    This is a delegate function. It doesn't nothing but invoke the correct
```

Comment:
'It _does_ nothing but invoke the correct ...' I know you didn't add this but you can correct this since you have made changes here.